sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Framework/Validators/RequireNonDefaultAttribute.cs

```diff
@@ -27,7 +28,14 @@ public override bool IsValid(object value)
             }
 
             var type = value.GetType();
```

Comment:
I'd suggest moving this down into the `else` - it's not needed until there.

Reply:
Done